about.html

```diff
@@ -12,62 +12,91 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
-      <footer>
+  <header>
+    <!--Nav-->
+    <nav>
+        <div class="logo">
+            <img src="/img/lambda-black.png"></i>
+        </div>
+        <div class="links">
+            <a href="/index.html">Home</a>
+            <a href="/about.html">About</a>
+            <a href="#products">Products</a>
+            <a href="#blog">Blog</a>
+            <a href="#contact">Contact</a>
+    </nav>
+            <div class="jumbotron2">
+                
+                </div>
+                <div class="jumbotext">
+                  <h1>About</h1>
+                <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+
+
+                </div>
+            </header>
+          </div>
+          <!--Gallery 1-->
+        <section class="gallery">
```

Comment:
Good use of semantic html here.  Don't forget to change your alt="strategy" for each area.